### PR TITLE
replace obsolete `exists` function with blazing new `access`

### PR DESCRIPTION
**Pull request by jeetiss**
https://twitter.com/IAmTrySound/status/1334520595376181250?s=20

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import * as os from 'os';
-import {promisify} from 'util';
 
-const fsExistsAsync = promisify(fs.exists);
 const fsReadFileAsync = fs.promises.readFile;
+const fsExistsAsync = (path: fs.PathLike): Promise<boolean> =>
+    fs.promises
+        .access(path, fs.constants.F_OK)
+        .then(() => true)
+        .catch(() => false);
```
Comment (TrySound) [inline src/index.ts:10]: Quite easy to do inline. Btw this mode is default.
```
try {
  await fs.promises.access(path)
} catch {
  continue;
}
```

Comment (TrySound) [reply src/index.ts:10]: Same with existsSync

**Comment by antonk52** [comment]
Thank you @jeetiss , the sync version still [contains](https://github.com/antonk52/lilconfig/pull/12/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R307) the `fs.existsSync` call, do you mind addressing it in your PR also? 🙂

**Review by antonk52 — APPROVED** [review]
Thanks @jeetiss 👍

**Review by TrySound — APPROVED** [review]

**Comment by antonk52** [comment]
This change has been released in [v2.0.2](https://github.com/antonk52/lilconfig/releases/tag/v2.0.2)

**Pull request merged**